openshift/catalogd/manifests/20-role-openshift-catalogd-metrics.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Very cool 👍 
we will need to have the same for operator-controller as well 👍

Reply:
Another nit:
For this one pass in the CI we need to add a commit message like 
`UPSTREAM: <carry>: MESSAGE`

That is because of the bumper
How we sync the things from upstream to downstream
And to allow the tool/bumper to be aware that this needs to be carried on.

Reply:
hmm, so no OCPBUGS-49410 at all? or should it be included in the message?
or for this we just use UPSTREAM carry and then for 4.19 for example we can do OCPBUGS-49410?

Reply:
4.19, 4.20 will be automatic synced 
Then, when you cherry-pick for 4.18 you would do like `UPSTREAM: PRID: Message` you will need to change that.

Reply:
I'm leaving the title as-is, 4.18 changes would require additional work in order to work and therefore should be synced automatically. If (given internal discussions) we want to make those changes for 4.18, it'd need to happen in a separate PR.